libraries/Microsoft.Bot.Builder.AI.LUIS/LuisApplication.cs

```diff
@@ -61,11 +55,11 @@ public LuisApplication(string applicationId, string endpointKey, string azureReg
         public string EndpointKey { get; set; }
 
         /// <summary>
-        /// Gets or sets Azure Region where endpoint is located.
+        /// Gets or sets LUIS endpoint.
         /// </summary>
         /// <value>
-        /// Azure Region where endpoint is located.
+        /// LUIS endpoint where application is hosted.
         /// </value>
-        public string AzureRegion { get; set; }
+        public string Endpoint { get; set; }
```

Comment:
Given that this is set in the constructor, consider making this "get-only" and removing the setter. No reason to change this once the object is constructed.